services/bookmark-message.service.js

```diff
@@ -9,7 +10,7 @@ class BookmarkMessageService {
       await BookmarkMessageService.#sendToUser(user, { embeds: [messageEmbed] });
     } catch (error) {
       if (error.code === RESTJSONErrorCodes.CannotSendMessagesToThisUser) {
-        const botSpamPlaygroundChannelId = '513125912070455296';
+        const { botSpamPlaygroundChannelId } = config.channels;
```

Comment:
This should be `config.channels.botSpamPlaygroundChannelId` not `config.channels`; right now its referring to the entire object. Also is there any reason to declare this variable instead of just referring to the config directly one line later?

Reply:
> This should be config.channels.botSpamPlaygroundChannelId not config.channels; right now its referring to the entire object

Not sure what this means, the relevant field is being destructured

> Also is there any reason to declare this variable instead of just referring to the config directly one line later?

Not really, just wanted to keep the PR changes to minimum LOC. I don't have opinions either way

Reply:
> > This should be config.channels.botSpamPlaygroundChannelId not config.channels; right now its referring to the entire object
> 
> Not sure what this means, the relevant field is being destructured
> 
Yeah, my mistake. Its apparently still too early for this. Is there a reason however you are destructing it there instead of accessing the value with dot notation as is happening with the rest of the references to `config.channels`? (eg `config.channels.noPointsChannelIds` in points.js). To me at least it feels weird to use that this once.

Reply:
ESLint told me to 😆

Reply:
Oh well in that case all hail our Lord and Savior ESLint 😆. I do think like this it looks cleaner however